src/SerializerInterface.php

```diff
@@ -22,6 +24,7 @@ public function serialize(\Closure $closure);
      * @param string $serialized Serialized closure.
      *
      * @return \Closure Unserialized closure.
+     * @throws ClosureUnserializationException if unserialization fails.
```

Comment:
This is something' that's implied by PSR-5 in their code examples. I'm also developing docblock fixers for php-cs-fixer that enforce the order param, throws, return for the annotations.

https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/887